Ginger/GingerCore/ALM/JIRA/Bll/JiraExportManager.cs

```diff
@@ -99,31 +99,36 @@ public JiraExportManager(JiraRepository.JiraRepository jiraRep)
         /// <returns></returns>
         string CheckIfDefectExist(string summary)
         {
-            //Search for exist based on summary
-            WhereDataList filterData = new WhereDataList();
-            filterData.Add(new WhereData() { Name = "summary", Values = new List<string>() { summary }, Operator = WhereOperator.And });
-
-            //and based on status, fetch statuses from JiraSettings.json 
-            DefectStatusColl jiraDefectStatuses = null;
-            JiraRepository.Helpers.JiraHelper jiraHelper = new JiraRepository.Helpers.JiraHelper();
-            if (jiraHelper.TryGetJiraDefectStatuses(out jiraDefectStatuses))
-            {
-                foreach (DefectStatus defectStatus in jiraDefectStatuses)
-                {
-                    ComparisionOperator comparisionOperator = GetComparisionOperator(defectStatus.Operator.ToLower());
-                    filterData.Add(new WhereData() { Name = "Status", Values = defectStatus.Status.ToList(), Operator = WhereOperator.And, ComparisionOperator = comparisionOperator });
-                }
-            }
-
-            var issues = jiraRepObj.GetJiraIssues(ALMCore.DefaultAlmConfig.ALMUserName, ALMCore.DefaultAlmConfig.ALMPassword, ALMCore.DefaultAlmConfig.ALMServerURL, ALMCore.DefaultAlmConfig.ALMProjectKey, ResourceType.DEFECT, filterData);
-            if (issues.DataResult.Count > 0)
-            {
-                return issues.DataResult.FirstOrDefault().key.ToString();
-            }
-            else
-            {
-                return "";
-            }
+            // Olego - 9/23/20 - temporary marking
+            // to check with Arvind
+
+            ////Search for exist based on summary
+            //WhereDataList filterData = new WhereDataList();
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6280018)